### PR TITLE
Feat/wt 2644 add disabled styles to native widgets

### PR DESCRIPTION
**Pull request by jordanch**
Extend native widget styling for disabled states.

### CODE REVIEW

File: styles/native/ts/core/widgets/textbox.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { font, input, spacing } from "../variables";
+import {  font, input, spacing } from "../variables";
```
Comment (jordanch) [inline styles/native/ts/core/widgets/textbox.ts]: nit: whitespace

File: styles/native/ts/core/variables.ts [hunk 2]
```diff
@@ -157,11 +161,13 @@ button = merge(button, custom.button || {});
 let input: VariablesInput = {
     // Colors
     color: font.color,
+    colorDisabled: font.colorDisabled,
     errorColor: brand.danger,
     labelColor: font.color,
+    labelColorDisabled: font.color,
```
Comment (jordanch) [inline styles/native/ts/core/variables.ts]: Is this meant to be the default value of `font.color`, what about `font.colorDisabled`?

File: styles/native/ts/core/widgets/buttons.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { button, contrast, font } from "../variables";
+import {  button, contrast, font } from "../variables";
```
Comment (jordanch) [inline styles/native/ts/core/widgets/buttons.ts]: nit: whitespace

File: styles/native/ts/core/widgets/buttons.ts [hunk 4]
```diff
@@ -26,20 +26,19 @@ export const ActionButton: ActionButtonType = {
         paddingVertical: button.paddingVertical,
         paddingHorizontal: button.paddingHorizontal,
     },
-    // todo: all disabled colors => move to variables?
     containerDisabled: {
         // All ViewStyle properties are allowed
-        backgroundColor: "#CED0D3",
-        borderColor: "#CED0D3",
+        backgroundColor: button.primary.borderColorDisabled,
```
Comment (jordanch) [inline styles/native/ts/core/widgets/buttons.ts]: I think this value should be: `button.primary.backgroundColorDisabled`

File: styles/native/ts/core/widgets/buttons.ts [hunk 5]
```diff
@@ -26,20 +26,19 @@ export const ActionButton: ActionButtonType = {
...
+        borderColor: button.primary.backgroundColorDisabled,
```
Comment (jordanch) [inline styles/native/ts/core/widgets/buttons.ts]: I think this value should be: `button.primary.borderColorDisabled`

File: styles/native/ts/core/variables.ts (hunk 2 above)
Comment (wegiswes) [reply styles/native/ts/core/variables.ts]: I thought the designs said it was the same color as font.color, but after double checking it's actually slightly different. I added a new var `font.labelColorDisabled` for it

**Review by wegiswes — APPROVED** [review]

**Pull request merged**